src/coreclr/pal/src/exception/remote-unwind.cpp

```diff
@@ -1422,25 +1422,50 @@ StepWithCompactNoEncoding(const libunwindInfo* info)
 
 #if defined(TARGET_ARM64)
 
-inline static bool
+#define ARM64_SYSCALL_OPCODE 0xd4001001
+
+static bool
+StepWithCompactNoEncoding(const libunwindInfo* info)
+{
+    // Check that the function is a syscall "wrapper" and assume there is no frame and pop the return address.
```

Comment:
The compact encoding value is 0 which from everything I've looked (docs, libunwind, lldb) at means "stop" unwinding.  lldb unwinds this by falling back to a unwind plan that "returns" (pc = lr) when the compact encoding plan fails. I wanted to play it safe by only doing this for syscall wrappers instead of any function with encoding == 0.  This is what I did for x64.  I don't have enough info or examples to open this up to all leaf nodes with encoding == 0.

So hopefully it is ok for now to minimize the risk and limit this to syscall wrappers.